api/api_test.go

```diff
@@ -797,6 +797,7 @@ func TestWebsocketClientDispatchEventWithoutMutatingData(t *testing.T) {
 			"status": "success",
 			"resource": {"url": "cart/1234"}
 			},
+			"externalType": "checkout_ui_extension",
```

Comment:
I think it's better to set an `external_type` [here](https://github.com/Shopify/shopify-cli-extensions/blob/2c51e0c296677e3ddafae6b78a7c069d1565518f/api/testdata/extension.config.yml#L10 ) so that we can properly assert that it's served correctly